message/infrastructure/sql/subscriber.go

```diff
@@ -64,13 +70,19 @@ func (c SubscriberConfig) validate() error {
 	if c.SchemaAdapter == nil {
 		return errors.New("schema adapter is nil")
 	}
+	if c.OffsetsAdapter == nil {
+		return errors.New("offsets adapter is nil")
+	}
 
 	return nil
 }
 
 // Subscriber makes SELECT queries on the chosen table with the interval defined in the config.
 // The rows are unmarshaled into Watermill messages.
 type Subscriber struct {
+	consumerIdBytes  []byte
```

Comment:
this is innovation for the future: it can be used for making active master HA model (without changing contract)